apps/web/pages/trials/edit/[id].tsx

```diff
@@ -91,7 +108,11 @@ const EditorEditTrialPage = () => {
       dispatch(selectedTrialGroupId(trialGroupName));
 
       editTrialObject = {...editTrialObject, ...ctml_json}
+      let priordata = {
+        prior_treatment_requirements : priorTreatmentRequirementNames
+      }
```

Comment:
this variable is not used

Reply:
I am using variable below to change the schema. EditTrialObject = {...editTrialObject, ...priordata}

Reply:
yes my mistake, I see it now